app/assets/javascripts/admin/tasktype/task_type_list_view.js

```diff
@@ -93,6 +93,18 @@ class TaskTypeListView extends React.PureComponent<Props, State> {
     });
   };
 
+  renderPlaceholder() {
+    return (
+      <React.Fragment>
+        {"There are no task types. You can "}
+        <Link to="/taskTypes/create">add a task type</Link>
+        {
+          " which can be used to configure certain properties for classes of tasks, for example, a description."
```

Comment:
`such as` instead of `for example` sounds nicer in my opinion. Maybe move it to the front as well: `... configure certain properties, such as a description, for classes of tasks.`